frontend/pages/_app.tsx

```diff
@@ -24,7 +24,7 @@ function MyApp({ Component, pageProps }: AppProps) {
               <meta property="og:title" content="Giving Coupons" />
               <meta property="og:type" content="website" />
               <meta property="og:url" content={PUBLIC_URL} />
-              <meta property="og:image" content={`${PUBLIC_URL}/giving-coupons.png`} />
+              <meta property="og:image" content={`${PUBLIC_URL}/logo-full.png`} />
```

Comment:
This image has to be 1200 pixels x 627 pixels

https://www.kapwing.com/resources/what-is-an-og-image-make-and-format-og-images-for-your-blog-or-webpage/